core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -143,12 +143,21 @@
     @Setter(AccessLevel.NONE)
     protected Info dockerDaemonInfo = null;
 
-    /*
+    /**
      * Set during container startup
+     * // TODO make it private
+     *
+     * @deprecated use {@link ContainerState#getContainerId()}
      */
     @Setter(AccessLevel.NONE)
+    @Deprecated
     protected String containerId;
 
+    /**
+     * Set during container startup
+     *
+     * @deprecated use {@link GenericContainer#getContainerInfo()}
+     */
```

Comment:
No `@Deprecated` annotation on the field?

Reply:
Good catch, let me add it

Reply:
done 👍